src/lib/tooltip/_tooltip-theme.scss

```diff
@@ -21,4 +23,10 @@ $mat-tooltip-vertical-padding: ($mat-tooltip-target-height - $mat-tooltip-font-s
     padding-top: $mat-tooltip-vertical-padding;
     padding-bottom: $mat-tooltip-vertical-padding;
   }
+
+  @media ($mat-xsmall) {
```

Comment:
Rather than using a media query in the css, please use `BreakpointObserver` from the cdk in the `MatTooltop` class to apply a mobile class (something like `.mat-tooltip-mobile`).

Reply:
That sounds like a better way, yes. I will close this PR and make another one.